### PR TITLE
Enhance the Logsviewer to support CLI to spin up logsviewer and import must-gather

### PR DESCRIPTION
**Pull request by machadovilaca**

### CODE REVIEW

File: deployment/elk_pod_template.yaml [hunk 1]
```diff
@@ -304,6 +304,6 @@ parameters:
 - description: Storage class to use
   value: ocs-storagecluster-ceph-rbd
   name: STORAGE_CLASS
-- description: Image tag
-  value: devel
-  name: IMAGE_TAG
+- description: LogsViewer image
+  value: quay.io/vladikr/logsviewer:devel
```
Comment (vladikr) [inline deployment/elk_pod_template.yaml:320]: we should move the default image out of my quay repo, at some point

**Review by vladikr — APPROVED** [review]
Thanks @machadovilaca !
Looks good.

**Comment by vladikr** [comment]
> @machadovilaca could you please add --signoff to your commits?

hmm... not sure what wasn't signed.

File: deployment/elk_pod_template.yaml (hunk 1 above)
Comment (machadovilaca) [reply deployment/elk_pod_template.yaml:320]: as long as you have the repository in your account I think you should own the default image
but maybe an option is [GitHub Packages](https://github.com/features/packages), which is free for open-source projects, making the image linked to the repository, and we could build and push it there on every release/main branch merge

**Pull request merged**